examples/local/cluster.tmpl.yaml

```diff
@@ -40,6 +40,7 @@ spec:
         insecurePort: 30010
         securePort: 30011
         domain: "ingress.${CLUSTER_NAME}.${COMMON_DOMAIN_INGRESS}"
+        wildcardDomain: "*.${CLUSTER_NAME}.${COMMON_DOMAIN_INGRESS}"
```

Comment:
We create a wildcard record to make it easier to test the ingress. Both records are in `clustertpr` which makes the domain names a bit more flexible.